src/tests/test_dte_data_models.py

```diff
@@ -862,34 +950,45 @@ def test_init_fail_fecha_ref_out_of_range(self) -> None:
 
         obj = self.obj_1
 
-        with self.assertRaises(pydantic.v1.ValidationError) as assert_raises_cm:
+        with self.assertRaises(pydantic.ValidationError) as assert_raises_cm:
             dataclasses.replace(
                 obj,
                 fecha_ref=date(2002, 7, 31),
             )
         self.assertEqual(
-            assert_raises_cm.exception.errors(),
+            assert_raises_cm.exception.errors(
+                include_context=False,
+                include_input=False,
+                include_url=False,
+            ),
             [
                 {
                     'loc': ('fecha_ref',),
-                    'msg': '("The date \'fecha_ref\' must be after 2002-08-01 and '
+                    'msg': 'Value error, ("The date \'fecha_ref\' must be after 2002-08-01 and '
                     'before 2050-12-31", datetime.date(2002, 7, 31))',
                     'type': 'value_error',
-                }
+                },
```

Comment:
Revert change to reduce diff size (unless the comma was added by Black).

```suggestion
                }
```